src/components/entity/ha-chart-base.html

```diff
@@ -335,8 +370,46 @@
           this._drawLegend();
         }
         this.resizeChart();
+        this.updateZoomlimits();
       }
     }
+    updateZoomlimits() {
+      if (!this._chart) return;
+      if (this.isTimeline) return;
+
+      const buffer = {
+        x: { min: null, max: null },
+        y: { min: null, max: null }
+      };
+      Object.entries(this._chart.scales).forEach(([name, scale]) => {
```

Comment:
Object.entries will break our ES5 build. Please don't use.

Reply:
google suggests `Object.keys()` is fine for ES5?
And I dont need the `if` below it anymore then?